learnAndCodeFinalProject/src/main/java/org/learnAndCode/Admin.java

```diff
@@ -0,0 +1,146 @@
+package org.learnAndCode;
+
+import org.learnAndCode.Database.DBConnection;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Admin extends User {
+
+    public Admin(String username, int roleId) {
+        super(username, roleId);
+    }
+
+    public static void displayMenu(PrintWriter writer, BufferedReader reader) throws IOException {
```

Comment:
displayMenu() doesn't fit into this class, it doesn't seems to be responsibility of Admin class. Same comment for other claases as well, Chef, Employee